ogr/services/github/project.py

```diff
@@ -248,72 +237,13 @@ def _get_collaborators_with_permission(self) -> dict:
         return collaborators
 
     def get_issue_list(self, status: IssueStatus = IssueStatus.open) -> List[Issue]:
-        issues = self.github_repo.get_issues(
-            state=status.name, sort="updated", direction="desc"
-        )
-        try:
-            return [
-                self._issue_from_github_object(issue)
-                for issue in issues
-                if not issue.pull_request
-            ]
-        except UnknownObjectException:
-            return []
-
-    def __get_issue(self, number: int) -> GithubIssue:
-        issue = self.github_repo.get_issue(number=number)
-        if issue.pull_request:
-            raise GithubAPIException(f"Requested issue #{number} is a pull request")
-        return issue
-
-    def get_issue_info(self, issue_id: int) -> Issue:
-        issue = self.__get_issue(number=issue_id)
-        return self._issue_from_github_object(issue)
+        return GithubIssue.get_list(self, status)
 
-    def _get_all_issue_comments(self, issue_id: int) -> List[IssueComment]:
-        issue = self.__get_issue(number=issue_id)
-        return [GithubIssueComment(raw_comment) for raw_comment in issue.get_comments()]
-
-    def issue_comment(self, issue_id: int, body: str) -> IssueComment:
-        """
-        Create comment on an issue.
-
-        :param issue_id: int The ID of the issue
-        :param body: str The text of the comment
-        :return: IssueComment
-        """
-        github_issue = self.__get_issue(number=issue_id)
-        comment = github_issue.create_comment(body)
-        return GithubIssueComment(comment)
+    def get_issue(self, issue_id: int) -> Issue:
+        return GithubIssue.get(self, issue_id)
```

Comment:
What about calling by name:

```suggestion
        return GithubIssue.get(project=self, issue_id=issue_id)
```

What about renaming `issue_id` to `id` in both places?